backend/question-service/src/controllers/questionController.ts

```diff
@@ -77,6 +85,7 @@ export const createQuestion = async (
       question: formatQuestionIndivResponse(newQuestion, newQuestionTemplate),
```

Comment:
are we returning the testcases? same for the api to get a question

Reply:
oops yes i wanted to created a pr asap so i don't block any tasks, i'll update the rest of the crud operations~